app/controllers/catalog_controller.rb

```diff
@@ -265,6 +265,8 @@ class CatalogController < ApplicationController
     config.add_sort_field 'relevance', sort: 'score desc, sortable_cho_title_ssi asc', label: 'Relevance'
     config.add_sort_field 'title', sort: 'sortable_cho_title_ssi asc, sortable_cho_creator_ssi asc', label: 'Title'
     config.add_sort_field 'creator', sort: 'sortable_cho_creator_ssi asc', label: 'Creator'
+    config.add_sort_field 'date_old_to_new', sort: 'cho_date_norm_min_isi asc', label: 'Date (old to new)'
```

Comment:
```suggestion
    config.add_sort_field 'date_old_to_new', sort: 'cho_date_norm_min_isi asc, sortable_cho_title_ssi asc', label: 'Date (old to new)'
```

Just a thought.

Reply:
probably a good thought

Reply:
(and... why we didn't think that thought when we added `creator`... 🤷‍♂️ )